mixpanel_query/client.py

```diff
@@ -886,8 +886,12 @@ def get_export(self, start_date, end_date, event=None, where=None, bucket_id=Non
             },
             response_format
         )
-        for line in response:
-            yield json.loads(_totext(line))
+        response_data = response.read()
+        lines = response_data.split('\n')
```

Comment:
comment added, let me know if that's what you had in mind or would like something else